### ISSUE TITLE
Intermittent reboots

### ISSUE THREAD

**Issue by hhezart** [J2Kbr/GtunerIV#16]
Increase in intermittent reboots since the last FW update, almost once an hour.  I'm using a power supply and a difference cable. This issue has been also been reported by Scachi.

**Comment by J2Kbr** [comment]
Please check if, by removing the sd-card, the intermittent reboots stops. thanks.

**Comment by hhezart** [comment]
I haven't been able to crash T2 in two days! No power supply, with SD card, with new cable (supplied cable's connector is flimsy and disconnects too easily). I've only managed to get two PS4 device disconnects but NO crash.  I've refactored my code a bit but I don't think that was the problem.

Update: I spoke too soon. Last night after 3 hours the device became very unstable and controller was getting disconnected every few minutes. I've removed the SD for now and will give it a go tonight.
